src/Data/Form.purs

```diff
@@ -243,15 +158,19 @@ lastName =
 
 active :: Field Expr
 active =
-  { name: val_ "Active"
-  , visibility: val_ true
-  , description: val_ "Is user's account active"
+  { name: string_ "Active"
+  , visibility: boolean_ true
+  , description
   , key: "active"
   , input: Toggle
-    { default: Just (val_ false)
+    { default: Just (boolean_ true)
     , value:
       { source: Nothing
       , value: Nothing
       }
     }
   }
+  where
+  description = if_ (equal_ (boolean_ true) (lookup_ "active"))
```

Comment:
I wonder if we can drop the `equal_` stuff? It should be semantically the same as `if_ (lookup_ "active")`, right?
```suggestion
  description = if_ (lookup_ "active")
```